modules/nw-configure-ingress-access-logging.adoc

```diff
@@ -116,3 +116,46 @@ spec:
   logging:
     access: null
 ----
+
+Allowing IngressController to modify the HAProxy log length when using a sidecar.
+
+Use `spec.logging.access.destination.syslog.maxLength` if you are using `spec.logging.access.destination.type: Syslog`.
+
++
```

Comment:
@jmanthei If you are following the formatting as the rest of the module, consider using the unordered list (`*`) (highlighted in green). If  you are not using a list, then you may want to remove the list continuation (`+`) (highlighted in orange) 
![image](https://github.com/openshift/openshift-docs/assets/74918891/594b64a2-de2f-4234-be69-dad26dccc807)